docker/Dockerfile.non_root

```diff
@@ -64,14 +64,24 @@ RUN pip install PyJWT==2.9.0 --no-cache-dir
 # Build Admin UI
 RUN chmod +x docker/build_admin_ui.sh && ./docker/build_admin_ui.sh
 
-# Generate prisma client
+### Prisma Handling for Non-Root #################################################
+# Prisma allows you to specify the binary cache directory to use 
 ENV PRISMA_BINARY_CACHE_DIR=/app/prisma
-RUN mkdir -p /.cache
-RUN chmod -R 777 /.cache
+
+# Set the TMPDIR environment variable, when this does not exist prisma raises "Error: ENOENT: no such file or directory, lstat '/var/folders'""
+ENV TMPDIR=/tmp 
+RUN mkdir -p /tmp && chmod 1777 /tmp
+
 RUN pip install nodejs-bin
 RUN pip install prisma
+
+# Make a /non-existent folder and assign chown to nobody
+RUN mkdir -p /nonexistent && chown -R nobody:nogroup /nonexistent
```

Comment:
@rajatvig could you clarify why you needed `/nonexistent` ?

Reply:
You'd need to execute the `prisma generate` as `nobody` and move the `chmod ...` instructions before.